<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WateringGUI</class>
 <widget class="QMainWindow" name="WateringGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WateringGUI</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QTabWidget" name="errorTab">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="homeTab">
       <attribute name="title">
        <string>Home</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>256</width>
          <height>256</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="wateringresource.qrc">:/img/logo.png</pixmap>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>10</y>
          <width>271</width>
          <height>161</height>
         </rect>
        </property>
        <property name="title">
         <string>Software Information</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Version:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>None</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Uptime:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="uptime">
             <property name="text">
              <string>None</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_13">
           <item>
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string>Active Devices:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="activeDev">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_14">
           <item>
            <widget class="QLabel" name="label_20">
             <property name="text">
              <string>Total Devices:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="totalDev">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="groupBox_6">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>180</y>
          <width>291</width>
          <height>231</height>
         </rect>
        </property>
        <property name="title">
         <string>Connected Devices</string>
        </property>
        <widget class="QWidget" name="verticalLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>271</width>
           <height>201</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QListWidget" name="mDevList"/>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="controlTab">
       <attribute name="title">
        <string>Control</string>
       </attribute>
       <widget class="QWidget" name="horizontalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>201</width>
          <height>31</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Device:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cDevList"/>
         </item>
        </layout>
       </widget>
       <widget class="QFrame" name="cFrame">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>791</width>
          <height>401</height>
         </rect>
        </property>
        <widget class="QGroupBox" name="groupBox">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>261</width>
           <height>211</height>
          </rect>
         </property>
         <property name="title">
          <string>Action List</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QPushButton" name="forceIrrbtn">
              <property name="styleSheet">
               <string notr="true">QPushButton {
    background-color: #1C1C1C;
	color: white;
    border-style: outset;
    border-width: 2px;
    border-color: #424242;
}
QPushButton:pressed {
    background-color: #A4A4A4;
    border-style: inset;
}</string>
              </property>
              <property name="text">
               <string>Force irrigation</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="checkHumbtn">
              <property name="styleSheet">
               <string notr="true">QPushButton {
    background-color: #1C1C1C;
	color: white;
    border-style: outset;
    border-width: 2px;
    border-color: #424242;
}
QPushButton:pressed {
    background-color: #A4A4A4;
    border-style: inset;
}</string>
              </property>
              <property name="text">
               <string>Check Current Humidty</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="restartDvcbtn">
              <property name="styleSheet">
               <string notr="true">QPushButton {
    background-color: #1C1C1C;
	color: white;
    border-style: outset;
    border-width: 2px;
    border-color: #424242;
}
QPushButton:pressed {
    background-color: #A4A4A4;
    border-style: inset;
}</string>
              </property>
              <property name="text">
               <string>Restart Device</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QGroupBox" name="controlGrp">
         <property name="geometry">
          <rect>
           <x>280</x>
           <y>10</y>
           <width>331</width>
           <height>58</height>
          </rect>
         </property>
         <property name="title">
          <string>Control</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>New Required Humidty:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="newReqHum"/>
            </item>
            <item>
             <widget class="QPushButton" name="setHum">
              <property name="styleSheet">
               <string notr="true">QPushButton {
    background-color: #1C1C1C;
	color: white;
    border-style: outset;
    border-width: 2px;
    border-color: #424242;
}
QPushButton:pressed {
    background-color: #A4A4A4;
    border-style: inset;
}</string>
              </property>
              <property name="text">
               <string>Set Humidity</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QGroupBox" name="advusr">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>280</x>
           <y>70</y>
           <width>332</width>
           <height>58</height>
          </rect>
         </property>
         <property name="title">
          <string>Advanced Users</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Custom Command:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="customCMD"/>
            </item>
            <item>
             <widget class="QPushButton" name="sendCMD">
              <property name="styleSheet">
               <string notr="true">QPushButton {
    background-color: #1C1C1C;
	color: white;
    border-style: outset;
    border-width: 2px;
    border-color: #424242;
}
QPushButton:pressed {
    background-color: #A4A4A4;
    border-style: inset;
}</string>
              </property>
              <property name="text">
               <string>Send</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QGroupBox" name="infoBox">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>230</y>
           <width>321</width>
           <height>161</height>
          </rect>
         </property>
         <property name="title">
          <string>Information</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Last Update:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="lastUpdate">
                <property name="text">
                 <string>None</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>Device ID:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="devID">
                <property name="text">
                 <string>None</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>Required Humidty:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="reqHum">
                <property name="text">
                 <string>None</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_17">
              <item>
               <widget class="QLabel" name="label_24">
                <property name="text">
                 <string>Last Humidty:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="lastHum">
                <property name="text">
                 <string>None</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QGroupBox" name="groupBox_7">
         <property name="geometry">
          <rect>
           <x>280</x>
           <y>130</y>
           <width>331</width>
           <height>91</height>
          </rect>
         </property>
         <property name="title">
          <string>Update Time Value(minutes)</string>
         </property>
         <widget class="QSlider" name="timeInterval">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>40</y>
            <width>160</width>
            <height>19</height>
           </rect>
          </property>
          <property name="minimum">
           <number>5</number>
          </property>
          <property name="maximum">
           <number>240</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>240</x>
            <y>40</y>
            <width>47</width>
            <height>13</height>
           </rect>
          </property>
          <property name="text">
           <string>None</string>
          </property>
         </widget>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="statTab">
       <attribute name="title">
        <string>Statistics</string>
       </attribute>
       <widget class="QWidget" name="horizontalLayoutWidget_11">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>201</width>
          <height>31</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_15">
         <item>
          <widget class="QLabel" name="label_22">
           <property name="text">
            <string>Device:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="sDevList"/>
         </item>
        </layout>
       </widget>
       <widget class="QFrame" name="sFrame">
        <property name="geometry">
         <rect>
          <x>-1</x>
          <y>39</y>
          <width>781</width>
          <height>391</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>521</width>
           <height>331</height>
          </rect>
         </property>
         <property name="title">
          <string>Last 7 Days</string>
         </property>
        </widget>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="geometry">
          <rect>
           <x>540</x>
           <y>30</y>
           <width>221</width>
           <height>331</height>
          </rect>
         </property>
         <property name="title">
          <string>irrigation time(s)</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_16">
          <item>
           <widget class="QListWidget" name="listWidget_2"/>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Error</string>
       </attribute>
       <widget class="QFrame" name="frame">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>761</width>
          <height>401</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>0</y>
           <width>751</width>
           <height>401</height>
          </rect>
         </property>
         <property name="title">
          <string>Error messages</string>
         </property>
         <widget class="QListWidget" name="errorView">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>40</y>
            <width>531</width>
            <height>261</height>
           </rect>
          </property>
         </widget>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="copyright">
      <property name="text">
       <string>© Sinus Automated Systems</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="wateringresource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>timeInterval</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>label_2</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>459</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>554</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
